firebase-database/ktx/ktx.gradle

```diff
@@ -46,6 +46,7 @@ dependencies {
 
     implementation project(':firebase-common')
     implementation project(':firebase-common:ktx')
+    implementation project(':firebase-components')
```

Comment:
I *think* taking these dependencies directly will lead to the it being included in the pom file?
If we want to continue being able to release components depending on an old version of common, we should remove this and bring the dep in transitively via a project dep on just common?

Reply:
`Failed to release project ':firebase-firestore'. Some of its dependencies don't have a released version: [com.google.firebase:firebase-components]`

Reply:
We would also have to change https://github.com/firebase/firebase-android-sdk/pull/918/files#diff-ca85ccc77c0711a2ebabca587683e98a from `implementation` to `api` to allow gradle builds to continue working

Reply:
good point, changed